aikau/src/main/resources/alfresco/layout/AlfTabContainer.js

```diff
@@ -290,12 +306,24 @@ define(["dojo/_base/declare",
             // we'll ensure that the first tab is both selected and will be immediately rendered
             var tabSelected = false;
             array.forEach(this.widgets, function(widget) {
-               if (widget.delayProcessing !== false && !widget.selected)
+               if ((widget.delayProcessing === true && !widget.selected) || 
+                   widget.delayProcessing === false)
```

Comment:
What happens if it's marked as false and isn't marked as selected? The conditional blcok wouldn't be entered